crates/sui-framework/packages/dwallet-system/sources/eth_dwallet.move

```diff
@@ -39,8 +44,22 @@ module dwallet_system::eth_dwallet {
         dwallet_id: vector<u8>,
         data_slot: u64,
         proof: vector<u8>,
+        latest_ethereum_state: &LatestEthereumState,
```

Comment:
```suggestion
```
I'm not sure that we care it is latest

Reply:
It is still needed since it has the contract address & data slot

Reply:
Ok, but then you must verify that `LatestEthereumState` and `EthereumState` match the same network